testsuite_alp.Dockerfile

```diff
@@ -1,18 +1,22 @@
-#
-# To test using this Dockerfile
-# Build Container Image: docker build -t netatalk-test:latest .
 
-# Init Container and Netatalk: docker run -d --network host --cap-add=NET_ADMIN --volume "<path to local folder>:/mnt/afpshare" --volume "<path to local folder>:/mnt/afpbackup" --env AFP_USER=test --env AFP_PASS=test --env SHARE_NAME='File Sharing' --env AFP_VERSION=7 --env AFP_GROUP=afpusers --env TZ=Europe/London --env INSECURE_AUTH=true --name netatalk-test netatalk-test:latest
+# Build Container Image: docker build --no-cache -f testsuite_alp.Dockerfile -t netatalk-test:latest .
```

Comment:
Just as a note: these code comments would be a great fit for your future lantest wiki page! And then we may not need them here in the dockerfile anymore?

Also, I wonder if it would make sense to have a paragraph about the testing docker images in the [DOCKER readme](https://github.com/Netatalk/netatalk/blob/main/DOCKER.md)? To make them more discoverable. Case in point, you didn't know about them before I pointed it out. ;)

Reply:
Done